src/constants.json

```
@@ -0,0 +1,16 @@
+{
+    "random": "Check this box to enable random safety car events.",
+    "random_max_occ": "The maximum number of times a safety car event could potentially occur (not guaranteed); applies to only random safety cars.",
```

```suggestion
    "random_max_occ": "The maximum number of times a random safety car event could potentially occur (not guaranteed).",
```
A suggestion to make this more succinct here.